chat-widget/src/components/livechatwidget/common/initWebChatComposer.ts

```diff
@@ -63,6 +65,17 @@ export const initWebChatComposer = (props: ILiveChatWidgetProps, state: ILiveCha
 
     if (!webChatStore) {
 
+        const addConversationalSurveyTagsCallback = (action: any) => {
+            const inMemoryState = executeReducer(state, { type: LiveChatWidgetActionType.GET_IN_MEMORY_STATE, payload: null });
+            const isConversationalSurvey = inMemoryState.appStates?.isConversationalSurvey;
+            if (isConversationalSurvey) {
+                if (!action.payload.activity.channelData.tags.includes(Constants.c2ConversationalSurveyMessageTag)) {
```

Comment:
null check  and  array type check before calling includes. 
in the invoker function, seems the tags could be null
<img width="1118" height="469" alt="image" src="https://github.com/user-attachments/assets/af77d3a3-2b81-441b-a720-73092832c4b7" />

Reply:
@Erli-ms I don't believe the additional check is necessary within `addConversationalSurveyTagsCallback`, since the `else` statement in `channelDataMiddleware` will handle the initialization of `action.payload.activity.channelData.tags` already.
<img width="622" height="231" alt="image" src="https://github.com/user-attachments/assets/611093d8-2950-40cc-8495-94921363c3e3" />